app/controlplane/integrations/dependencytrack/cyclonedx/v1/api.proto

```diff
@@ -22,28 +22,19 @@ import "validate/validate.proto";
 
 // RegistrationRequest is what the integration expect as input of the pre-registration step
 message RegistrationRequest {
-	RegistrationConfig config = 1 [(validate.rules).message.required = true];
+	string domain = 1 [(validate.rules).string.min_len = 1];
```

Comment:
Should we add proper validations here: 

```
string domain = 1 (validate.rules).string.address = true];
```

 or

```
string domain = 1 (validate.rules).string.uri = true];
```

Reply:
that probably makes sense, good catch! Will change it!